src/ResourceManager/Cdn/Commands.Cdn/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Current Release
 
+## Version 5.0.6
```

Comment:
remove the version header, leave the change description directly under `Current Release`

Reply:
I'll fix the test. Thanks!